app/app/src/main/java/org/omecproject/up4/impl/Up4DeviceManager.java

```diff
@@ -1181,6 +1292,7 @@ public void run() {
         }
 
         private void checkStateAndReconcile() throws UpfProgrammableException {
+            // TODO: reconcile meters!!
```

Comment:
Reconciliation seems important, is this something you want to address in this PR or a future one?

Reply:
Doing it on a subsequent PR. For now, we replicate the state via the meter manager events.